### ISSUE TITLE
RaspAC

### ISSUE THREAD

**Issue by hariskhalid** [tpudlik/RaspAC#1]
Hi,@tpudlik 
                    I hope you are fine. I have one question regarding the lirc.conf file created using the raw codes for your lg ac. How did you find the gap while recording in raw mode. kindly let me know thanks

**Comment by tpudlik** [comment]
There are two different questions I think you might be asking.  Let me try to answer both.

1. How did I determine the number on [line 14](https://github.com/tpudlik/RaspAC/blob/master/lircd.conf#L14) of my `lircd.conf`?  I created the config using [an existing config from the LIRC website](http://lirc.sourceforge.net/remotes/solidtek/ACK-581) as a template.  That config had a gap of "28205", and I just kept the value.

2. How to deal with an error in `irrecord` in which the program claims it's unable to determine the signal gap?  This problem is caused a difference in the structure of signals sent by TV remotes (for which `irrecord` was designed) and AC remotes. As discussed by [Peter Li](https://web.archive.org/web/20170201131003/http://absurdlycertain.blogspot.com/2013_03_01_archive.html), the latter send much longer commands containing a description of the complete state of the device, rather than merely a short specification of the desired change.

    My solution was the same as Peter Li’s:
* I used `mode2 `to record the pulse train emitted by the remote when a button is pressed, piping the output to a text file. The `-m` option of `mode2` was useful here, as it strips extraneous descriptors from the output, encoding in a format that can be used immediately in the next step.
*  I created a config file `lircd.conf` using the raw codes obtained in the previous step. I took one of the raw config files available on the LIRC website as a template.

The `lircd.conf` file obtained in this way works just as well as one generated by `irrecord`.
